core/src/main/java/com/datastax/oss/driver/api/core/tracker/RequestTracker.java

```diff
@@ -40,28 +40,73 @@
    *     GenericType) session.execute} call until the result is made available to the client).
    * @param executionProfile the execution profile of this request.
    * @param node the node that returned the successful response.
+   * @deprecated override {@link #onSuccess(Request, long, DriverExecutionProfile, Node, String)}
+   *     instead.
    */
+  @Deprecated
   default void onSuccess(
       @NonNull Request request,
       long latencyNanos,
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node) {}
 
+  /**
+   * Invoked each time a request succeeds.
+   *
+   * @param latencyNanos the overall execution time (from the {@link Session#execute(Request,
+   *     GenericType) session.execute} call until the result is made available to the client).
+   * @param executionProfile the execution profile of this request.
+   * @param node the node that returned the successful response.
+   * @param requestPrefix the dedicated log prefix for a request
```

Comment:
```suggestion
   * @param requestPrefix the dedicated log prefix for this request
```
(same as in the description for executionProfile)